index.js

```diff
@@ -36,29 +36,252 @@ const authors = {
  */
 const packages = {
   'babel-core': 'https://opencollective.com/babel',
-  bower: 'https://opencollective.com/bower',
-  chai: 'https://opencollective.com/chaijs',
-  cheerio: 'https://opencollective.com/cheerio',
-  choo: 'https://opencollective.com/choo',
-  gulp: 'https://opencollective.com/gulpjs',
+  'bower': 'https://opencollective.com/bower',
+  'chai': 'https://opencollective.com/chaijs',
+  'cheerio': 'https://opencollective.com/cheerio',
+  'choo': 'https://opencollective.com/choo',
+  'gulp': 'https://opencollective.com/gulpjs',
   'gulp-cli': 'https://opencollective.com/gulpjs',
-  hoodie: 'https://opencollective.com/hoodie',
-  koa: 'https://opencollective.com/koajs',
+  'hoodie': 'https://opencollective.com/hoodie',
+  'koa': 'https://opencollective.com/koajs',
   'material-ui': 'https://opencollective.com/material-ui',
-  mocha: 'https://opencollective.com/mochajs',
-  parcel: 'https://opencollective.com/parcel',
-  phenomic: 'https://opencollective.com/phenomic',
-  preact: 'https://opencollective.com/preact',
-  pug: 'https://opencollective.com/pug',
+  'mocha': 'https://opencollective.com/mochajs',
+  'parcel': 'https://opencollective.com/parcel',
+  'phenomic': 'https://opencollective.com/phenomic',
+  'preact': 'https://opencollective.com/preact',
+  'pug': 'https://opencollective.com/pug',
   'react-native-elements': 'https://opencollective.com/react-native-elements',
   'redux-devtools-extension': 'https://opencollective.com/redux-devtools-extension',
-  rollup: 'https://opencollective.com/rollup',
+  'rollup': 'https://opencollective.com/rollup',
   'socket.io': 'https://opencollective.com/socketio',
   'styled-components': 'https://opencollective.com/styled-components',
-  tachyons: 'https://opencollective.com/tachyons',
-  webpack: 'https://opencollective.com/webpack',
-  yo: 'https://opencollective.com/yeoman',
-  levelup: 'https://opencollective.com/level'
+  'tachyons': 'https://opencollective.com/tachyons',
+  'webpack': 'https://opencollective.com/webpack',
+  'yo': 'https://opencollective.com/yeoman',
+  'levelup': 'https://opencollective.com/level',
+  'sierra-library': 'https://opencollective.com/sierra',
+  'vue': 'https://opencollective.com/vuejs',
+  'sketchpad': 'https://opencollective.com/sketchpad',
+  'slim-js': 'https://opencollective.com/slimjs',
+  'ng2-date-picker': 'https://opencollective.com/angular-datepicker',
+  'esdiscuss.org': 'https://opencollective.com/esdiscuss',
+  'undefined': 'https://opencollective.com/turf',
```

Comment:
This looks like an error!

I'm guessing that this is caused by their root `package.json` not having a `"name"` field. See here: https://github.com/Turfjs/turf/blob/master/package.json

I think this is actually pretty common for mono-repo projects. https://github.com/Turfjs/turf/blob/master/packages/turf/package.json

I wonder if this happened more than once in your script. If that happened, then you'd have overwritten the `'undefined'` field multiple times and many projects may have not made it into this dump. Thoughts?

Reply:
Ah good catch. Yes just remove that line. If they don’t have a name in their package.json there is no way you can do anything with it anyway. So not much we can do here.